test/metric_dimension/metrics_append_dimension_test.go

```diff
@@ -46,7 +46,8 @@ func getTestRunners(env *environment.MetaData) []*test_runner.TestRunner {
 	if testRunners == nil {
 		factory := dimension.GetDimensionFactory(*env)
 		testRunners = []*test_runner.TestRunner{
-			{TestRunner: &NoAppendDimensionTestRunner{Base: test_runner.BaseTestRunner{DimensionFactory: factory}}},
+			{TestRunner: &NoAppendDimensionTestRunner{test_runner.BaseTestRunner{DimensionFactory: factory}}},
```

Comment:
Think the append dimension tests could be added to this test suite?

Reply:
The append dimension tests would cover a lot though. There are 2 use cases that would need to test:
* The global append dimension
* The custom dimension which would only exist for some plugin
This would be an outside scope so yeah maybe something to consider.

Reply:
Oh we totally can. I just had added one test case that was the easiest to check bare minimum of does it actually not add anything. Zhenyu had assigned Thomas to work on adding Append Dimension tests since he implemented the feature, but I don't think he got to it. I will talk to Zhenyu

Reply:
yeah...I have a pending task to add that but didn't get time to push the PR. And yes the plan is to add at least one more test here for append dimensions.
Just to confirm that this PR is for aggregation but not append dimension, right? (Since I see `append_dimensions` in the new config.json, but, if I understand it correctly, that's just a requirement to use `aggregation_dimensions`, right?)